src/main/java/org/openmrs/keycloak/provider/OpenmrsAuthenticatorProviderFactory.java

```diff
@@ -26,164 +40,142 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.net.URL;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Properties;
-
-public class OpenmrsAuthenticatorProviderFactory
-		implements UserStorageProviderFactory<OpenmrsAuthenticator> {
-
+public class OpenmrsAuthenticatorProviderFactory implements UserStorageProviderFactory<OpenmrsAuthenticator> {
+	
 	public static final String PROVIDER_NAME = "openmrs-authentication-provider";
-
+	
 	private static final Logger log = LoggerFactory.getLogger(OpenmrsAuthenticatorProviderFactory.class);
-
+	
 	private static final List<ProviderConfigProperty> CONFIG_METADATA;
-
+	
 	static {
-		CONFIG_METADATA = ProviderConfigurationBuilder.create()
-				.property()
-					.name("JDBC URL")
-					.defaultValue("mysql://localhost:3306/openmrs")
-					.helpText("The JDBC URL for the OpenMRS MySQL Server")
-					.type(ProviderConfigProperty.STRING_TYPE)
-					.add()
-				.property()
-					.name("User Name")
-					.defaultValue("openmrs")
-					.helpText("The user name of the MySQL user")
-					.type(ProviderConfigProperty.STRING_TYPE)
-					.add()
-				.property()
-					.name("Password")
-					.defaultValue("openmrs")
-					.helpText("The passsword for the MySQL user")
-					.type(ProviderConfigProperty.PASSWORD)
-					.secret(true)
-					.add()
-				.build();
+		CONFIG_METADATA = ProviderConfigurationBuilder.create().property().name("JDBC URL")
+		        .defaultValue("mysql://localhost:3306/openmrs").helpText("The JDBC URL for the OpenMRS MySQL Server")
+		        .type(ProviderConfigProperty.STRING_TYPE).add().property().name("User Name").defaultValue("openmrs")
+		        .helpText("The user name of the MySQL user").type(ProviderConfigProperty.STRING_TYPE).add().property()
+		        .name("Password").defaultValue("openmrs").helpText("The passsword for the MySQL user")
+		        .type(ProviderConfigProperty.PASSWORD).secret(true).add().build();
 	}
-
+	
 	@Override
 	public OpenmrsAuthenticator create(KeycloakSession keycloakSession, ComponentModel model) {
 		MultivaluedHashMap<String, String> config = model.getConfig();
-		EntityManagerFactory emf = new HibernatePersistenceProvider()
-				.createContainerEntityManagerFactory(new PersistenceUnitInfoImpl(), ImmutableMap.<String, Object>builder()
-						.put(AvailableSettings.JPA_JDBC_DRIVER, "com.mysql.jdbc.Driver")
```

Comment:
@ibacher Shouldn't this be
```
put(AvailableSettings.JPA_JDBC_DRIVER, "com.mysql.cj.jdbc.Driver")
```

Reply:
Depends on which version of the MySQL connector is running. You’re right if we’re running 8.0 or newer, but my thought was we might be able to leverage the version bundled with Keycloak, which uses connector 5.1 where this is the right class name.

Reply:
@ibacher I tried running the server with both the versions and I still get this error

```
Uncaught server error: java.lang.NoClassDefFoundError: org/hibernate/jpa/HibernatePersistenceProvider
```

Any idea?